histomicstk/features/ComputeNucleiFeatures.py

```diff
@@ -167,6 +180,26 @@ def ComputeNucleiFeatures(im_label, im_nuclei, im_cytoplasm=None,
 
         feature_list.append(fgrad_cytoplasm)
 
+    # compute nuclei haralick features
+    if haralick_features_flag:
+
+        fharalick_nuclei = ComputeHaralickFeatures(im_label, im_nuclei,
```

Comment:
Deepak - dealing with the varied ranges of input images is a troubling problem. Either we stretch the ranges of inputs - which creates huge problems for inter-image comparability - or we adopt some assumptions and ask users to follow best practices in analysis & development:

1. int type intensity images range from 0-255
2. float type intensity images range from 0-1
3. color conversions, deconvolution etc. attempt to preserve these ranges
4. filter responses may deviate from these ranges - these are not strictly intensity images

Color normalization should be used always - this will map intensity values to a reasonable range where subsequent operations like segmentation and feature extraction can make some reasonable assumptions.